Content.Shared/Pinpointer/NavMapBeaconComponent.cs

```diff
@@ -5,15 +5,21 @@ namespace Content.Shared.Pinpointer;
 /// <summary>
 /// Will show a marker on a NavMap.
 /// </summary>
-[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[RegisterComponent, Access(typeof(SharedNavMapSystem))]
```

Comment:
Context for this change?

Reply:
in technical details

> unnetworked the beacon and its fields since client code never touched it and it has no reason to and it shouldnt be possible for client to know about it anyway (can undo if theres plans for prediction or something)

i also said it briefly in the commit message...

Reply:
Thank you.